packages/integrations/image/README.md

```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommending installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
```

Comment:
Nice! Love the revision.

Just a small typo here:

```suggestion
Along with our integration, we recommend installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
```

Reply:
Thanks for the help!  💜